### ISSUE TITLE
Implement backwards compatibility changes for URP 12

### ISSUE THREAD

**Issue by Zallist** [Zallist/unity-universal-shadergraph-extensions#5]
@[bmourit](https://github.com/bmourit) on issue https://github.com/Zallist/unity-universal-shadergraph-extensions/issues/3 has pointed out that this Simple Lit shader does NOT work on URP 12. This issue encompasses the work to getting it to work:

- `#if` checks in C# code so that BlendModePreserveSpecular is stripped before unity 2022
- keyword or safety check in hlsl to remove the blendmode code

**Comment by bmourit** [comment]
Great. I'll leave the current code & comment out differences between this as the PBRForwardPass/PBRGBufferPass for ease of updating. I've switched all to Unity's namespaces as well, for reasons you stated in the closed issue. I think I've found a simple solution to my GBuffer issue as well, and it keeps things in like with Unity pretty nicely.
Thanks again for doing this, it is really needed & much requested. I'm sure it will help many!
